core/src/main/java/org/apache/servicecomb/core/SCBEngine.java

```diff
@@ -223,10 +228,22 @@ public synchronized void init() {
         }
         status = SCBStatus.FAILED;
         throw new IllegalStateException("ServiceComb init failed.", e);
+      } finally {
+        ServiceInfoPrinter();
       }
     }
   }
 
+  private void ServiceInfoPrinter() {
```

Comment:
- method name should follow CamelCase
- please use verb as method name, e.g. `printServiceInfo` is better than `ServiceInfoPrinter`

Reply:
The name of the method have been changed to printServiceInfo.